CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Changelog
 
 This is a changelog for **ZachranObed** application.
+## [1.0.2]
```

Comment:
I guess this should be still 1.0.1 if we are merging it to  `release/1.0.1`?

Reply:
yes, I have edited it (originally I had it directed to 1.0.2). Thanks for the sharp eyes :)